namespaces/resource-namespace.md

```diff
@@ -0,0 +1,231 @@
+# Resource Namespace
+
+This document extends [Refract][] [MSON Namespace][] to define REST Resource data structure elements.
+
+## About this Document
+
+This document conforms to RFC 2119, which says:
+
+> The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+[MSON][] is used throughout this document.
+
+## Supporting Data Types
+
+### Href (string)
+
+The value of `Href` is to be used as a URI Template, as defined in RFC 6570.
```

Comment:
I think it would be nice if these RFC numbers was hyperlinks to the underlying specification.